src/query/api/v1/handler/graphite/render.go

```diff
@@ -200,6 +209,10 @@ func (h *renderHandler) serveHTTP(
 		SortApplied: true,
 	}
 
+	if !exhaustive {
+		w.Header().Set(handler.LimitHeader, "true")
```

Comment:
nit: Should we make this a constant?  i.e. `w.Header().Set(handler.LimitHeader, handler.LimitHeaderSeriesLimitApplied)`?

In the future if we limit the results due to a datacenter/remote region being down, we can concat these together perhaps.

ie.
```go
w.Header().Set(handler.LimitHeader, handler.LimitHeaderSeriesLimitApplied)

// in handler pkg
const LimitHeaderSeriesLimitApplied = "max_fetch_series_limit_applied"
```

Then in future if we have another type:
```
w.Header().Set(handler.LimitHeader, handler.LimitHeaderValue(handler.LimitHeaderSeriesLimitApplied, ...))

// in handler pkg
func LimitHeaderValues(str ...string) string {
  return strings.Join(str..., ",")
}
```

Reply:
The future is now!